### PR TITLE
Fix deploy issues

### PR DESCRIPTION
**Pull request by shaahmad**
This should fix most, if not all, deploy issues. To apply to your respective staging target, copy and adjust `tufte/serverSettings.json` and `tufte/clientSettings.json` and maybe delete your `bakedSite` to rebake pages.

- [x] duplicate some client settings and server settings
- [x] have serverside code use serverSettings and clientside code use clientSettings
- [x] fix configuration issues in the serverSettings.json file on staging stemming from env => json conversion
- [x] fix wordpress db connection issues
- [x] fix wrong webpack url for prod/staging builds

### CODE REVIEW

**Comment by MarcelGerber** [comment]
Another thing we need to do for every server we change over from `master` to `next` is to run `yarn startAdminServer` and `yarn startDeployQueueServer` in `pm2`, instead of the `serve` and `deploy-queue` it currently runs (and which no longer exist).
I didn't do that yet for tufte because I don't know much about `pm2`, but after we do that tufte.owid.cloud should be accessible again.

**Comment by breck7** [comment]
Nice! LGTM. I forget, do we explain in
the settings/ readme how to use these settings?

That would be nice to add if we dont have it yet.

**Review by MarcelGerber — COMMENTED** [review]
Nice! I have one suggestion (you decide what's best to there), but other than that this is great!

I also noticed another issue (in fact, the reason why tufte.owid.cloud doesn't work), but I'm working on that and will put up another PR for that later on.

File: settings/serverSettings.ts [hunk 1]
```diff
@@ -10,6 +10,39 @@ const localOverrides: any = fs.existsSync(absoluteSettingsPath)
     ? fs.readJsonSync(absoluteSettingsPath)
     : {}
 
+export const ADMIN_SERVER_PORT = localOverrides.ADMIN_SERVER_PORT ?? 3030
```
[the remark this answers is not in the document]
Comment (shaahmad) [reply settings/serverSettings.ts:13]: I did consider that! However, it seemed less straightforward/more confusing to me that some settings in a file explicitly named clientSettings would then be used on the server-side. Considering that these env files are unlikely to change that often, I'm not sure we will run into much drift.

My doomed quest to remove upstream usage of these env variables was somewhat fruitful, and I plan on merging that soon after I merge this. I think we can get rid of at least half of the duplicated variables in the client side just by using window.href, etc.

**Pull request merged**